elastic/kibana
# [Security Solution][Entity Analytics] Only request read privileges for risk score in Entity Analytics dashboard

We previously required both READ and WRITE privileges for merely displaying risk score data within the Entity Analytics dashboard.

This PR ensures people with only READ access can still view the risk data.
The missing privileges call out will now point out only READ privilege is required:

<img width="1482" alt="Screenshot 2024-05-16 at 14 59 31" src="https://github.com/elastic/kibana/assets/2423976/927105b9-facc-41d1-8ee4-34b69f69edc6">



## How to test

1. Create a role which has access to all security related indices and Security Permissions.
2. Ensure that the user only has READ permissions for the `.risk-score*` index
3. Create a new user with said role from step 1
4. Login with that user and navigate to **Security -> Dashboards -> Entity Analytics**
